src/components/events-components/EventsTitle.js

```diff
@@ -7,11 +7,11 @@ const EventsTitle = () => {
   return (
     <>
       <div
-        className="w-full min-h-screen"
+        className="w-full min-h-[135vh]"
```

Comment:
If we want the events title to not cut off the vector animations we need the individual vectors on the bottom. The solution wouldn't be to make the events title bigger to 135vh. This leads to poor UI.

Reply:
I know, didn't realize we had the vector files until after I finished